src/mixxx.cpp

```diff
@@ -698,6 +701,15 @@ void MixxxMainWindow::finalize() {
         qWarning() << "Central widget was not deleted by our sendPostedEvents trick.";
     }
 
+    // Delete Controls created by skins
+    for (ControlObject* pControl : m_skinCreatedControls) {
+        VERIFY_OR_DEBUG_ASSERT(pControl) {
```

Comment:
This assertion is not needed. The whole loop could be replaced by [qDeleteAll()](https://doc.qt.io/qt-5/qtalgorithms.html#qDeleteAll)